google/cloud/bigquery/client.py

```diff
@@ -2113,7 +2113,8 @@ def load_table_from_dataframe(
 
             Due to the way REPEATED fields are encoded in the ``parquet`` file
             format, a mismatch with the existing table schema can occur, and
-            100% compatibility cannot be guaranteed for REPEATED fields.
+            100% compatibility cannot be guaranteed for REPEATED fields when
+            using the parquet format.
```

Comment:
This could mislead people to think that REPEATED fields are supported for CSV (not that it would make any sense). Perhaps we need a second note describing that only scalar data types are supported with CSV?

Reply:
@tswast I updated the note to try to make that clear.